mkposters/post_install.py

```diff
@@ -0,0 +1,53 @@
+import subprocess
+
+
+def main():
+    bash_cmd = """
+    # Detect the compute architecture (linux-arm64, macos-arm64, macos-x64) and download the appropriate distribution release of dart-sass
+    # https://github.com/sass/dart-sass/releases/tag/1.50.1
+
+    sass_release="1.50.1"
+    arch=$(uname -m)
+    kernel=$(uname -s)
+
+    echo Detected architecture: $arch and kernel: $kernel.
+
+    if [[ "$arch" =~ ^(arm64|aarch64)$ ]] && [ "$kernel" = "Linux" ]; then
+        curl -sL "https://github.com/sass/dart-sass/releases/download/${sass_release}/dart-sass-${sass_release}-linux-arm64.tar.gz" > dart.tar.gz && tar -xzf dart.tar.gz && rm dart.tar.gz
+    elif [ "$arch" = "x86_64" ] && [ "$kernel" = "Linux" ]; then
+        curl -sL "https://github.com/sass/dart-sass/releases/download/${sass_release}/dart-sass-${sass_release}-linux-x64.tar.gz" > dart.tar.gz && tar -xzf dart.tar.gz && rm dart.tar.gz
+    elif [ "$arch" = "arm64" ] && [ "$kernel" = "Darwin" ]; then
+        curl -sL "https://github.com/sass/dart-sass/releases/download/${sass_release}/dart-sass-${sass_release}-macos-arm64.tar.gz" > dart.tar.gz && tar -xzf dart.tar.gz && rm dart.tar.gz
+    elif [ "$arch" = "x86_64" ] && [ "$kernel" = "Darwin" ]; then
+        curl -sL "https://github.com/sass/dart-sass/releases/download/${sass_release}/dart-sass-${sass_release}-macos-x64.tar.gz" > dart.tar.gz && tar -xzf dart.tar.gz && rm dart.tar.gz
+    else
+        echo "Unsupported architecture: $arch and kernel: $kernel"
+    fi
+
+    # move the dart-sass binary to the correct location
+    mkloc=$(pip show mkposters | grep Location | cut -d ' ' -f 2)
```

Comment:
Nit: I think a cleaner/safer way of finding this is to evaluate `pathlib.Path(__file__).resolve().parent` and then interpolate this into the command.